src/overlays/actors/ovl_En_Tru/z_en_tru.c

```diff
@@ -228,37 +233,36 @@ void func_80A85AA4(EnTruUnkStruct* arg0, Vec3f* arg1, f32 arg2, f32 arg3, f32 ar
 
 void func_80A85BCC(EnTruUnkStruct* arg0, PlayState* play) {
     s32 i;
+    GraphicsContext* gfxCtx = play->state.gfxCtx;
```

Comment:
```suggestion
    GraphicsContext* gfxCtx = play->state.gfxCtx;
    s32 i;
```
Can these be swapped (applies to all functions).